test/E2E/Feedback.purs

```diff
@@ -0,0 +1,31 @@
+-- | Store and retrieve values in simple JS variable (window.ctlTestFeedback),
+-- | in order to establish a communication between the Examples and E2E tests.
+-- | Retrieval must be called from Puppeteer, while publishing happens in the browser.
+module Test.E2E.Feedback
+  ( publishTestFeedback
+  , retrieveTestFeedback
+  , testFeedbackIsTrue
+  ) where
+
+import Effect (Effect)
+import Effect.Aff (Aff)
+import Effect.Class (liftEffect)
+import Foreign (Foreign, unsafeFromForeign)
+import Prelude
```

Comment:
Pleas move this to the first line of the import list, and add a space between it and the other imports